### PR TITLE
Add new telemetry implementation that sends events to r.stripe.com

### PR DESCRIPTION
**Pull request by gracegoo-stripe**
* To send additional dimensions, updated stripeClient to set accountId and cliVersion in the workspaceState
* Created a temporary telemetry instance that will write to both sources
* Removed ip as a dimension

## Testing 
Verified events are sent to r.stripe.com via server side logs. Added some unit testing around version check and authentication

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -409,8 +411,10 @@
     "os-name": "^3.1.0",
     "proxyquire": "^2.1.3",
     "ps-list": "^7.1.0",
-    "public-ip": "^4.0.1",
+    "query-string": "^7.0.0",
```
Comment (vcheung-stripe) [inline package.json:414]: Nit: is this dependency needed? Node has a builtin `querystring` module: https://nodejs.org/api/querystring.html#querystring_querystring_stringify_obj_sep_eq_options

**Review by vcheung-stripe — APPROVED** [review]
This looks great!

File: package.json (hunk 1 above)
Comment (gracegoo-stripe) [reply package.json]: Oh thank you! Removed the dependency.

File: src/telemetry.ts [hunk 2]
```diff
@@ -58,7 +72,7 @@ export class GATelemetry implements Telemetry {
     return this._isTelemetryEnabled;
   }
 
-  async setup() {
+  setup() {
     if (!this.isTelemetryEnabled) {
```
Comment (vcheung-stripe) [inline src/telemetry.ts:76]: I think this needs parens:

```suggestion
    if (!this.isTelemetryEnabled()) {
```

Comment (gracegoo-stripe) [reply src/telemetry.ts]: Ah, I thought already added that but apparently I did not! Good catch, updated!

**Review by vcheung-stripe — APPROVED** [review]
LGTM

**Pull request merged**